zap/src/main/java/org/parosproxy/paros/control/Control.java

```diff
@@ -666,4 +670,47 @@ public void setMode(Mode mode) {
             model.getOptionsParam().getViewParam().setMode(mode.name());
         }
     }
+
+    public static LinkedList<String> getRecentSessionList() {
+        File file = new File(Constant.getZapHome(), "recent.txt");
+        LinkedList<String> recentSessionList = new LinkedList<>();
+        try {
+            file.createNewFile();
+            Scanner s = new Scanner(file);
+            while (s.hasNextLine()) {
+                recentSessionList.add(s.nextLine());
+            }
+            s.close();
+        } catch (IOException e) {
+            log.error(e.getMessage(), e);
+        }
+        return recentSessionList;
+    }
+
+    public static void setRecentSessionList(String path) {
+        File recentFile = new File(Constant.getZapHome(), "recent.txt");
```

Comment:
I dont think we should have more files cluttering up the home dir ;) This can go in a list in the standard config file. Although we usually manage things in this file via the Options screens we dont actually have to do that - for example the Mode is in that file but has its own control on the toolbar.

Reply:
Good idea